src/SFA.DAS.ApprenticeCommitments/Infrastructure/ApprenticeCommitmentsHealthCheck.cs

```diff
@@ -0,0 +1,32 @@
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using SFA.DAS.Api.Common.Infrastructure;
+using SFA.DAS.ApprenticeCommitments.Application.Services;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Infrastructure
+{
+    public class ApprenticeCommitmentsHealthCheck : IHealthCheck
+    {
+        private const string HealthCheckResultDescription = "Employer Incentives Api Health Check";
+        private readonly ApprenticeCommitmentsService service;
+
+        public ApprenticeCommitmentsHealthCheck(ApprenticeCommitmentsService service)
```

Comment:
Should we not be injecting the Interface into the service

Reply:
See https://github.com/SkillsFundingAgency/das-apim-endpoints/pull/232#discussion_r556401338